include/open62541/client_subscriptions.h

```diff
@@ -78,6 +82,24 @@ UA_Client_Subscriptions_create_async(UA_Client *client,
     UA_ClientAsyncServiceCallback callback,
     void *userdata, UA_UInt32 *requestId);
 
+UA_CreateSubscriptionResponse UA_EXPORT UA_THREADSAFE
```

Comment:
This needs a lot more inline documentation in the header.
Please add it. What does the method do, what are intended use cases, why is it not possible to achive the same with the other API methods.

Reply:
I added some documentation